### ISSUE TITLE
Aspirational: Create view(s) more appropriate to manage large scale k8s operations

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#11285]
Large scale k8s operations might face some challenges with managing their k8s infrastructures. Thinking of Edge devices is a scenario where we might have thousands of clusters of devices. 

Could we provide, at least, a better overview in those cases? How would that interface would look like? Create an extension or more to customise the UI to those particular usecases?

Also, scenarios where multi-tier Rancher might be employed due to sheer scale is an important factor to consider.

**Comment by aalves08** [comment]
Yes, in fact all of that performance work is going to make it much easier to manage k8s resources at scale with Dashboard.

The idea here is more in line of offering a different overview to those use cases, especially in a multi-tier scenario, where I believe the end user (depending on it's role) might benefit from rather than the traditional Rancher view. Abstracting states up the chain is a must for this to work, for sure.

I am thinking even in different overviews for different roles of k8s admin, which might be a plus for XL-k8s-management enterprises.

Food for thought after attending Susecon 🙏

**Comment by nwmac** [comment]
Going to close this as won't fix. Will wait until this is on the roadmap, then we can re-open with a clearer direction.
